django/bin/django-admin.py

```diff
@@ -1,5 +1,13 @@
 #!/usr/bin/env python
+import warnings
+
 from django.core import management
+from django.utils.deprecation import RemovedInDjango40Warning
```

Comment:
I think we should wrap this in a `try... except...` in order to catch any final lingerers: 

```
try: 
    from django.utils.deprecation import RemovedInDjango40Warning
except ImportError: 
    raise Exception(
        'django-admin.py was deprecated in Django 3.1 and removed in Django '
        '4.0. Please manually remove this script from your venv and use '
        'django-admin instead.'
    )
```

(Since a `pip install -U Django` isn't ever going to actually delete it.)

Reply:
Should we just put the warning above the Django imports then? And then use the builtin `DeprecationWarning`, but include text about Django 4.0?

Reply:
Meh... I think, if after 4.0, someone is still running this, we should fail hard. (Exception, rather than warning.) But open to arguments... — this is just the last safety net. I don't think we should spend too much time here.